Server/mods/deathmatch/logic/luadefs/CLuaObjectDefs.cpp

```diff
@@ -222,6 +225,11 @@ int CLuaObjectDefs::SetObjectScale(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaObjectDefs::IsObjectMoving(CObject* const pObject)
```

Comment:
Could you add `const` to the parameter itself?
```cpp
bool CLuaObjectDefs::BreakObject(const CObject* const pObject)
```

Reply:
No, because the methods in the CObject class are not constant and it would require editing many methods in this class because the IsMoving method is used by other functions